src/CONST.ts

```diff
@@ -633,6 +633,10 @@ const CONST = {
         INBOX: 'inbox',
     },
 
+    // TODO: change to expensify-policy once the new domain is available
+    EXPENSIFY_POLICY_DOMAIN: 'expensify_policy',
```

Comment:
Can you please change this to the dash format?
```suggestion
    EXPENSIFY_POLICY_DOMAIN: 'expensify-policy',
```

Reply:
@mountiny done!